source/includes/connection-snippets/x509-snippets/cpp-connection.cpp

```diff
@@ -0,0 +1,31 @@
+// begin x509 connection
+#include <iostream>
+#include <bsoncxx/builder/stream/document.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/client.hpp>
+#include <mongocxx/uri.hpp>
+#include <mongocxx/instance.hpp>
+#include <mongocxx/options/client.hpp>
+#include <mongocxx/options/ssl.hpp>
+#include <bsoncxx/builder/basic/kvp.hpp>
```

Comment:
I believe the following includes aren't necessary anymore.

- #include <bsoncxx/builder/stream/document.hpp>
- #include <bsoncxx/json.hpp>
- #include <bsoncxx/builder/basic/kvp.hpp>

Reply:
fixed